setup.py

```diff
@@ -13,13 +13,18 @@
 # limitations under the License.
 
 import base64
+import os
 import setuptools
 
 import streamer
 
 with open('README.md', 'r') as f:
   long_description = f.read()
 
+install_prerequisites = ['PyYAML']
+if os.name == 'nt':
```

Comment:
I think this will only run on the system where we build the package, which will usually be Linux.  Checking `os.name` here won't work, because no matter what platform we install it on later, this part is only run once when we build the package.

Instead, to express a platform-specific dependency, I just found out there is a syntax for that:

`'pypiwin32 ; platform_system=="Windows"'`

See https://stackoverflow.com/a/42946495